examples/ReAct/hotpotqa/README.md

```diff
@@ -0,0 +1,8 @@
+## Run
+For llama3
+```bash
+export PYTHONPATH=your/path/to/llm-reasoners:$PYTHONPATH
+CUDA_VISIBLE_DEVICES=0 python -m torch.distributed.run --nproc_per_node 1 examples/ReAct/hotpotqa/inference.py --base_lm llama3 --model_dir $LLAMA3_CKPTS --llama_size "8B" --batch_size 1 --temperature 0
+```
```

Comment:
Post some results.

Better to add the result of the simplest baseline (directly prompting LLM to answer)